src/main/js/page_objects/blueocean/bluePipelineActivity.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         activityTableEntries: 'table.activity-table tbody tr',
         runButton: 'a.run-button',
         toastOpenButton: {
-            selector: '//a[@class="action" and position()=1]',
+            selector: '//div[@class="toast"]/a[@class="action" and text()="Open"]',
```

Comment:
That test was failing for me all the time and it seemed like it was because the selector was not specific enough. All I can say is that it is working now without an error, so I think it's better to be more specific Vs being less specific and having flaky tests.